Samples/ResourceManagement/cpp-console-unpackaged/console_unpackaged_app.vcxproj

```diff
@@ -1,9 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="packages\Microsoft.ProjectReunion.WinUI.0.8.0-preview\build\native\Microsoft.ProjectReunion.WinUI.props" Condition="Exists('packages\Microsoft.ProjectReunion.WinUI.0.8.0-preview\build\native\Microsoft.ProjectReunion.WinUI.props')" />
-  <Import Project="packages\Microsoft.ProjectReunion.Foundation.0.8.0-preview\build\native\Microsoft.ProjectReunion.Foundation.props" Condition="Exists('packages\Microsoft.ProjectReunion.Foundation.0.8.0-preview\build\native\Microsoft.ProjectReunion.Foundation.props')" />
-  <Import Project="packages\Microsoft.ProjectReunion.DWrite.0.8.0-preview\build\Microsoft.ProjectReunion.DWrite.props" Condition="Exists('packages\Microsoft.ProjectReunion.DWrite.0.8.0-preview\build\Microsoft.ProjectReunion.DWrite.props')" />
-  <Import Project="packages\Microsoft.ProjectReunion.0.8.0-preview\build\native\Microsoft.ProjectReunion.props" Condition="Exists('packages\Microsoft.ProjectReunion.0.8.0-preview\build\native\Microsoft.ProjectReunion.props')" />
+  <Import Project="packages\Microsoft.WindowsAppSDK.1.0.0-preview1\build\native\Microsoft.WindowsAppSDK.props" Condition="Exists('packages\Microsoft.WindowsAppSDK.1.0.0-preview1\build\native\Microsoft.WindowsAppSDK.props')" />
+  <Import Project="packages\Microsoft.Windows.SDK.BuildTools.10.0.20348.19\build\Microsoft.Windows.SDK.BuildTools.props" Condition="Exists('packages\Microsoft.Windows.SDK.BuildTools.10.0.20348.19\build\Microsoft.Windows.SDK.BuildTools.props')" />
```

Comment:
What is this new "Microsoft.Windows.SDK.BuildTools" package?

Reply:
When adding WindowsAppSDK nuget package, it added it automatically. I assumed it was some kind of dependency, but I checked and the solution works correctly without it. I'll remove it :)

Reply:
It is a dependency from Microsoft.WindowsAppSDK. If I try to remove it, I am shown the following error:
```Unable to uninstall 'Microsoft.Windows.SDK.BuildTools.10.0.20348.19' because 'Microsoft.WindowsAppSDK.1.0.0-preview1' depends on it.```

Reply:
Microsoft.Windows.SDK.BuildTools was added to WAS recently. It removes the dependency on the Windows SDK installed on the machine for build tools like makepri.exe, makeappx.exe, etc.

Reply:
Thanks @rohanp-msft! :)